config/longrun_configs/longrun_aquaplanet_clearsky_1M.yml

```
@@ -0,0 +1,18 @@
+dt: "120secs"
+t_end: "100days"
+h_elem: 16
+z_elem: 63
+dz_bottom: 30.0
+dz_top: 3000.0
+z_max: 55000.0
+rayleigh_sponge: true
+moist: "equil"
+precip_model: "1M"
+surface_setup: "DefaultMoninObukhov"
+vert_diff: "FriersonDiffusion"
+implicit_diffusion: true
+approximate_linear_solve_iters: 2
+rad: "clearsky"
+dt_rad: "6hours"
+job_id: "gpu_aquaplanet_clearsky_1M"
```

```suggestion
job_id: "longrun_aquaplanet_clearsky_1M"
```
(This needs to be the same as JOB_NAME in the pipeline for the artifacts to be uploaded. I usually make it the same as the config file name. Also remember to add it to ci_plots and longrun docs.)